src/api/kube.ts

```diff
@@ -28,19 +28,21 @@ import { V1alpha2Certificate } from './typings/cert-manager'
 const AWAIT_TIMEOUT_S = 30
 
 export class KubeHelper {
-  kc = new KubeConfig()
-  portForwardHelper = new PortForward(this.kc, true)
-  logHelper = new Log(this.kc)
+  public static readonly KUBE_CONFIG = KubeHelper.initializeKubeConfig()
```

Comment:
I am not keen on the idea to make kubeconfig static.

Reply:
Why?

Reply:
`KubeHelper` is helper class which is used to communicate with Kubernetes API. By making kubeconfig static you basically make whole class static, so it becomes more a namespace than a class... It is my vision...